### PR TITLE
Moved API Files To A Better More Organized Location

### PR DESCRIPTION
**Pull request by RPSMain**
## Summary

Made API Into A Folder And Organized The File Names More
[Preview](https://cord.readthedocs.io/en/movers/api/)

### Changes

-   `improvements`: API File Locations

 ### Check off the following

-   [x] I have tested my changes with the current requirements
-   [x] My Code follows the pep8 code style.

### CODE REVIEW

**Comment by RPSMain** [comment]
categorization

File: docs/api/index.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+.. toctree::
+    :maxdepth: 1
+
+    pincer
+    core
+    commands
+    middleware
+    objects/index
+    utils
```
Comment (Sigmanificient) [inline docs/api/index.rst:14]: missing ref links

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: docs/api/index.rst (hunk 1 above)
Comment (RPSMain) [reply docs/api/index.rst:14]: Wdym it was like that’s before

Comment (Sigmanificient) [reply docs/api/index.rst:14]: oh wrong index xD

**Review by Sigmanificient — APPROVED** [review]

**Comment by Sigmanificient** [comment]
Reviewed everything within the preview, all is fine, no need to wait longer

**Pull request merged**